packages/alchemy/src/client/multiOwnerLightAccountClient.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  createMultiOwnerLightAccount,
+  multiOwnerLightAccountClientActions,
+  type CreateMultiOwnerLightAccountParams,
+  type MultiOwnerLightAccount,
+} from "@alchemy/aa-accounts";
+import type { HttpTransport, SmartAccountSigner } from "@alchemy/aa-core";
+import { custom, type Chain, type CustomTransport, type Transport } from "viem";
+import { AlchemyProviderConfigSchema } from "../schema.js";
+import { createAlchemySmartAccountClientFromRpcClient } from "./internal/smartAccountClientFromRpc.js";
+import { createAlchemyPublicRpcClient } from "./rpcClient.js";
+import {
+  type AlchemySmartAccountClient,
+  type AlchemySmartAccountClientConfig,
+} from "./smartAccountClient.js";
+
+export type AlchemyMultiOwnerLightAccountClientConfig<
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+> = Omit<
+  CreateMultiOwnerLightAccountParams<HttpTransport, TSigner>,
+  "transport" | "chain"
+> &
+  Omit<
+    AlchemySmartAccountClientConfig<
+      Transport,
+      Chain,
+      MultiOwnerLightAccount<TSigner>
+    >,
+    "account"
+  >;
+
+export async function createMultiOwnerLightAccountAlchemyClient<
+  TSigner extends SmartAccountSigner = SmartAccountSigner
+>(
+  params: AlchemyMultiOwnerLightAccountClientConfig<TSigner>
+): Promise<
+  AlchemySmartAccountClient<
+    CustomTransport,
+    Chain | undefined,
+    MultiOwnerLightAccount<TSigner>
+  >
```

Comment:
you need to specify the `actions` generic on this client or else you won't get the completions

Reply:
done